src/parse/errors.rs

```diff
@@ -413,68 +413,17 @@ impl Error {
         process::exit(0);
     }
 
-    #[doc(hidden)]
+    /// Writes the error message to the writer.
     pub fn write_to<W: Write>(&self, w: &mut W) -> io::Result<()> {
         write!(w, "{}", self.message)
     }
 
-    #[doc(hidden)]
-    pub fn group_conflict<O, U>(
```

Comment:
I don't get why this and some other functions below are removed.

Reply:
Because `cargo check` reported them to be unused, but they seemingly get used in certain macros, I'm bringing them back.

Reply:
For the record - `cargo check --tests` checks *tests* specifically.